blockchain/test-utils/src/lib.rs

```diff
@@ -118,9 +118,9 @@ pub fn get_blocks_with_recipients<R: RngCore + CryptoRng>(
         prev_block = Some(block.clone());
 
         let signature = make_block_signature(&block, rng);
-        let metadata = make_block_metadata(block.id.clone(), rng);
+        //let metadata = make_block_metadata(block.id.clone(), rng);
 
-        let block_data = BlockData::new(block, block_contents, signature, metadata);
+        let block_data = BlockData::new(block, block_contents, signature, None);
```

Comment:
actually, i want to revert this, I commented this out initially when the test was really slow to try to eliminate what might be the cause